### ISSUE TITLE
Page load fails sometimes, shows javascript missing error. Reload fixes this.

### ISSUE THREAD

**Issue by omahlama** [nailor/unpayhs#7]
The error that I found in console:

`Uncaught TypeError: Cannot read property 'instance' of undefined` in desktop_all.js:207

With a bit of inquiry I found that the offending command is:

```
$(function() { hs.paywall.instance.updateCountersInUI(); });
```

**Comment by nailor** [comment]
Hm, wonder when that gets triggered. Completely possible that the code derps because we override the instance with our special case.

There's at least a race condition here: https://github.com/nailor/unpayhs/blob/master/js/unpay.js#L22.
Given that no-one ever tried setting the `hs.paywall.instance` after we injected our triggers, we'll return `undefined` when it is accessed.

**Comment by nailor** [comment]
Fixed in #8
